terraform/aws/implementation/modules/eks/main.tf

```diff
@@ -257,27 +308,42 @@ resource "helm_release" "building_blocks" {
   #  Values needed for orchestration service
   set {
     name  = "fhirConverterUrl"
-    value = "https://k8s-phdiplayground${terraform.workspace}.${var.region}.elb.amazonaws.com/fhir-converter"
+    value = "https://${var.domain_name}/fhir-converter"
```

Comment:
Declaring and using this as a `domain_name` variable makes better sense, especially since there is talks to change the name from the phdiplayground at some point in the future.  It makes the code cleaner and easier to read as well.